### PR TITLE
fix(docs): update to valid flex basis value

### PR DESCRIPTION
**Pull request by nkrantz**
Quick PR to fix a bug on our Flex docs that someone filed on Issues (#2432)

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-website/src/pages/components/flex/index.mdx [hunk 1]
```diff
@@ -123,7 +123,7 @@ you use the Flex as a custom element, it is up to you to manage the resulting ac
 
 <LivePreview scope={{Box, Flex}} language="jsx">
   {`<Flex>
-    <Flex grow shrink basis>
+    <Flex grow shrink basis={0}>
```
Comment (andipants12) [inline packages/paste-website/src/pages/components/flex/index.mdx:126]: question: Did we consider setting this to a non-zero value? Previously, as the issue points out, the prop value for `basis` would have been `true`. Is there a benefit to using a truthy value here?

Comment (andipants12) [reply packages/paste-website/src/pages/components/flex/index.mdx:126]: question: Why is the `basis` prop on line 135 unchanged? Could we please check the docs page once more (and prob the stories) to ensure we catch other invalid prop values?

**Review by gloriliale — APPROVED** [review]
✅ question: why `{0}` as opposed to `{1}`?

**Review by andipants12 — COMMENTED** [review]
Left two questions, with potential requested changes. Generally looks good!

Question: also, curious if the prop definitions for `grow`, `shrink`, and `flex` have been reviewed? I see [here](https://github.com/twilio-labs/paste/pull/2433/files#diff-ced09f1009b61b8fc05a577963bcb45e52e18e48cf4d880139cf2d8d41d39991L328) that grow and shrink don't have the same primitive types. I'm wondering if that is intentional.

File: packages/paste-website/src/pages/components/flex/index.mdx (hunk 1 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/components/flex/index.mdx:126]: update from convo: I'll update all the `basis` props on this page to be valid values and I updated the AC of [this ticket](https://issues.corp.twilio.com/browse/DSYS-2903) to include adding more helpful info for the basis, grow, and shrink properties to the docs.

**Review by andipants12 — APPROVED** [review]

**Pull request merged**